Gateway/Config/PaymentPlans/PaymentPlanConfig.php

```diff
@@ -83,17 +84,18 @@ public static function defaultConfigForFeePlan(FeePlan $plan): array
     {
         $deferred_trigger_limit_days = $plan->getDeferredTriggerLimitDays();
         return [
+            'allowed' => $plan->allowed,
             'kind' => $plan->kind,
 
             'installmentsCount' => $plan->installments_count,
 
             'deferredDays' => intval($plan->deferred_days),
             'deferredMonths' => intval($plan->deferred_months),
 
-            'deferredTriggerEnable' => isset($deferred_trigger_limit_days),
-            'deferredTriggerDays' => $deferred_trigger_limit_days,
+            'deferredTriggerEnable' => !empty($deferred_trigger_limit_days),
+            'deferredTriggerDays' => intval($deferred_trigger_limit_days),
 
-            'enabled' => $plan->installments_count === 3,
+            'enabled' => 0,
```

Comment:
By default, all plans are disabled now ?

Are you sure this is a product request ?

Reply:
by default plan 3 installments is up .